changelog.md

```diff
@@ -1,6 +1,24 @@
+## Changes from 1.4.10 to 1.4.11
+
+### Fixed issues
+
+- [MARATHON-7992](https://jira.mesosphere.com/browse/MARATHON-7992) - Validation errors on root group in large clusters can lead Marathon to crash.
+
+
+## Changes from 1.4.9 to 1.4.10
+
+### Fixed issues
+
+- [MARATHON-7974](https://jira.mesosphere.com/browse/MARATHON-7974) - Fixes a security concern for a non super user. It was discovered that permissions for GroupResources was not honored and it is now resolved.
```

Comment:
I think that this is more alarming than it needs to be. It wasn't that we have a security vulnerability, but rather, only super users could access things. So, it was the opposite of a vulnerability (security was too tight).

Reply:
@kensipe can you please confirm that my understanding here is correct?

Reply:
That's how it is phrased on the releases page: https://github.com/mesosphere/marathon/releases
Should it be rephrased anyway?

Reply:
Yes, I think it should be. Also, I think we should fix the release notes.